packages/syft/src/syft/client/enclave_client.py

```diff
@@ -96,7 +96,19 @@ def request_code_execution(self, code: SubmitUserCode):
 
         apis = []
         for k, v in code.input_policy_init_kwargs.items():
-            api = APIRegistry.api_for(k.node_id, k.verify_key)
```

Comment:
Before we used to use the node uid and verify key of the domain owner to index the client.